packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/flush-archive.constants.js

```diff
@@ -0,0 +1,10 @@
+import {
+  COLD_ARCHIVE_STATES as STATES,
+  COLD_ARCHIVE_TRACKING_PREFIX as TRACKING,
+  REGION as DEFAULT_REGION,
+} from '../../cold-archives.constants';
+
+export const COLD_ARCHIVE_STATES = STATES;
+export const COLD_ARCHIVE_TRACKING_PREFIX = TRACKING;
+export const COLD_ARCHIVE_DEFAULT_REGION = DEFAULT_REGION;
```

Comment:
**nitpick:** Instead of import and exporting the existing constants, directly import from the parent file wherever it is required.

Reply:
Done.